assets/src/scss/new-identity/style.scss

```diff
@@ -9,4 +9,19 @@
   --block-columns--step-number--background: var(--gp-green-100);
   --block-columns--step-number--color: var(--grey-900);
   --block-columns--card-header--background: var(--p4-dark-green-800);
+  --button--focus--outline: 2px solid #99c3ff;
+  --button--focus--outline-offset: 2px;
+  --top-page-tags--tag-item--background: var(--gp-green-100);
+  --post-tag-button--font-weight: 700;
+  --post-tag-button--border-color: var(--gp-green-100);
+  --post-tag-button--hover--color: var(--grey-900);
```

Comment:
The border color should also be changed on hover, right now it looks a bit weird